src/events/acceleration.js

```diff
@@ -619,14 +619,14 @@ p5.prototype.setShakeThreshold = function(val) {
 
 p5.prototype._ondeviceorientation = function(e) {
   this._updatePRotations();
-  if (this._angleMode === constants.radians) {
-    e.beta = e.beta * (_PI / 180.0);
-    e.gamma = e.gamma * (_PI / 180.0);
-    e.alpha = e.alpha * (_PI / 180.0);
+
+  let multiplier = 1;
+  if (this._angleMode === constants.RADIANS) {
+    multiplier = constants.PI / 180.0;
   }
-  this._setProperty('rotationX', e.beta);
-  this._setProperty('rotationY', e.gamma);
-  this._setProperty('rotationZ', e.alpha);
+  this._setProperty('rotationX', e.beta * multiplier);
+  this._setProperty('rotationY', e.gamma * multiplier);
+  this._setProperty('rotationZ', e.alpha * multiplier);
```

Comment:
There is an internal method to convert angles, can you see if you can use that to simplify the implementation instead? https://github.com/processing/p5.js/blob/main/src/math/trigonometry.js#L437

Reply:
Ive added a new internal method, similar to the one you pointed out which converts from degrees to the current angle mode unit (since that one was missing and was needed here).
Another simplification i made is that in the `_handleMotion` method, instead of switching the equations based on angle mode, rotation properties are now converted into degrees beforehand and all calculations are done in degrees.